shared/src/commonMain/kotlin/co/nimblehq/blisskmmic/data/network/core/NetworkClient.kt

```diff
@@ -0,0 +1,58 @@
+package co.nimblehq.blisskmmic.data.network.core
+
+import co.nimblehq.jsonapi.json.JsonApi
+import io.ktor.client.HttpClient
+import io.ktor.client.call.*
+import io.ktor.client.engine.*
+import io.ktor.client.plugins.contentnegotiation.*
+import io.ktor.client.plugins.logging.*
+import io.ktor.client.request.*
+import io.ktor.client.statement.*
+import io.ktor.serialization.kotlinx.json.*
+import kotlinx.coroutines.flow.Flow
+import kotlinx.coroutines.flow.flow
+import kotlinx.serialization.json.Json
+
+class NetworkClient {
+
+    val client: HttpClient
+
+    val json = Json {
+        prettyPrint = true
+        isLenient = true
+        ignoreUnknownKeys = true
+    }
+
+    constructor(engine: HttpClientEngine? = null) {
+        if (engine == null) {
+            client = HttpClient() {
+                install(Logging)
+                install(ContentNegotiation) {
+                    json(Json {
+                        prettyPrint = true
+                        isLenient = true
+                        ignoreUnknownKeys = true
+                    })
```

Comment:
reusing `json` definition?
```suggestion
                    json(json)
```

Reply:
Fixed in https://github.com/blyscuit/kmm-swiftUI-IC/pull/51/commits/ed98f64ab85943583117ab7411cb6b14c82e3533